src/classes/Contest.ts

```diff
@@ -133,11 +141,13 @@ export class Contest {
 
     console.log('Contest.ts: currently selected: ' + this.currentlySelected + ', votesAllowed: ' + this.votesAllowed);
 
-    if (this.currentlySelected === this.votesAllowed) {
+    // eslint-disable-next-line
+    if (this.currentlySelected == this.votesAllowed) {
```

Comment:
_question_ these values are both numbers, so their type is already equivalent. Why not use triple equals here?